wger/nutrition/models.py

```diff
@@ -85,8 +85,8 @@ class Meta:
     has_goal_calories = models.BooleanField(verbose_name=_('Use daily calories'),
                                             default=False,
                                             help_text=_("Tick the box if you want to mark this "
-                                                        "plan as having a goal amount of calories."
-                                                        " You can use the calculator or enter the "
+                                                        "plan as having a goal amount of calories. "
+                                                        "You can use the calculator or enter the "
```

Comment:
Nothing has changed here

Reply:
There is a change in spacing which was a result of flake8 issuing errors in this file about specific lines. That's why they are also additional changes in the file.